package.json

```diff
@@ -58,6 +58,7 @@
     "gulp-util": "^3.0.7",
     "gzip-js": "~0.3.2",
     "jest": "^15.1.1",
+    "jest-cli": "^15.1.1",
```

Comment:
In terms of linting, would it be preferable to pull in the eslint updates from `master` and make `15-stable` conform?  Or update `15-stable` to conform to its current linting rules?  I seem to be getting linting errors on install due to [these changes](https://github.com/facebook/react/commit/4294a7c908d06de5f04d7b78a3e32c5721702ef6) and [this rule](https://github.com/facebook/react/blob/15-stable/.eslintrc.js#L44).  That rule was [removed in `16`](https://github.com/facebook/react/commit/32f5b034ed229d048f76ae74e18d270edc801dbf#diff-e4403a877d80de653400d88d85e4801aL46).